### ISSUE TITLE
Unable to Save child resources

### ISSUE THREAD

**Issue by frogabog** [modxbot/migrate#6580]
frogabog created Redmine issue ID 6580

Cannot save a child resource.  Save is highlighted, but does not work.  Ctrl+s doesn't work either.

**Comment by frogabog** [comment]
frogabog submitted:

Upon further investigation, I tried changing the parent resource after using the context menu to create a child.  When I selected the parent node from the tree, MODX tells me that the "resource can't be it's own parent".

The resources in question were duplicated, not original resources.  OK, so I thought maybe there is a bug that prevented duplicated resources modifications from saving. 

I attempted to create a fresh new document, then tell it to be a child.  This worked to set the parent, the above error did not occur.  However, save still does not work.
